src/pages/periodProduct.tsx

```diff
@@ -161,7 +161,7 @@ export const getServerSideProps = (ctx: NextPageContext): {} => {
     if (singleOperatorCookie) {
         const { selectedServices } = JSON.parse(singleOperatorCookie);
         props = {
-            zoneName: selectedServices.length > 1 ? 'Multiple Services' : selectedServices[0].lineName,
+            zoneName: selectedServices?.length > 1 ? 'Multiple Services' : selectedServices[0].lineName,
```

Comment:
Can `selectedServices` ever be null or empty? Since if it was `selectedServices[0].lineName` would error. If it never would be then we can remove the optional chaining